src/main/groovy/org/pih/warehouse/core/Constants.groovy

```diff
@@ -25,6 +25,10 @@ class Constants {
     static final adminControllers = ['createProduct', 'admin']
     static final adminActions = ['product': ['create'], 'person': ['list'], 'user': ['list'], 'location': ['edit'], 'shipper': ['create'], 'locationGroup': ['create'], 'locationType': ['create'], '*': ['delete']]
 
+    // TODO: Don't add more dates here! We should refactor all usages of these constants to instead use the
+    //       DateTimeFormatter constants in DateUtil. The backend should always return dates in the same format so that
+    //       the frontend can easily parse response objects. Let the frontend decide what the display format of each
+    //       individual field should be.
```

Comment:
my idea here is that we should always use the same format when returning dates to the frontend, and then let the frontend format the dates for display.

As such, I hope we can eventually remove most of these from the backend and simply call toString() on the Instant/LocalDate datatypes in our toJson() method.

As for how to parse on the frontend, [Alan did a discovery that is quite relevant](https://pihemr.atlassian.net/wiki/spaces/OB/pages/3107422231)